### DISCUSSION TITLE
Ender 3 S1 Jyers UBL coming soon ?

### DISCUSSION THREAD

**Discussion by yozza1412** [Jyers/Marlin#1919] [General Discussion]
would jyers have UBL firmware soon for the Ender 3 S1, the S1 is so internittent with Z offest issues,   i have UBL on other V2s and pray they have it for the S1 soon...   many users having same issues to a point the z offsets changes constantly and the CR touch seems to incorrectly read the corners wrong.....

**Comment by PapsDizzy** [comment]
when its done it will come

**Comment by dut97** [comment]
Any ETA on precompiled bin files for the S1?
